core/record/recordExprValueConverter.js

```diff
@@ -0,0 +1,60 @@
+import * as R from 'ramda'
+
+import * as Survey from '@core/survey/survey'
+import * as NodeDef from '@core/survey/nodeDef'
+import * as Record from '@core/record/record'
+import * as Node from '@core/record/node'
+
+const _valueExprToValueBoolean = (survey, record, nodeCtx, valueExpr) =>
+  R.is(String, valueExpr) && R.includes(valueExpr, ['true', 'false'])
+    ? valueExpr
+    : null
+
```

Comment:
is it possible that the expression returns a boolean and not a string?

Reply:
it can be, right